VERSION

```diff
@@ -1 +1 @@
-1.2.3
\ No newline at end of file
+1.2.4
```

Comment:
The version bump is usually handled by expeditor, but there was a glitch when the previous change was merged and the version did not get bumped. This is a one time manual change to be able to release the license scout change and fix chef-infra-server broken pipeline.